typed-racket-lib/typed-racket/rep/type-rep.rkt

```diff
@@ -508,6 +508,19 @@
       (= (length kws) 1)
       (equal? kws (sort kws Keyword<?))))
 
+;; a Rest argument description
+;; tys: the cycle describing the rest args
+;; e.g.
+;; tys = (list Number) means all provided rest args
```

Comment:
Does this have to be `(list T)` instead of `T`?

Reply:
that makes handling any rest args (i.e. uniform or more interesting cycles) all able to use the same code

Reply:
Ah, I see now